pagopa/io-app
# fix: [IOBP-31] Fix privacy button in IDPay beneficiary details screen

## Short description
This PR fixes the Preferences & Privacy button in the IDPay details screen allowing the user to navigate to the correct screen

https://github.com/pagopa/io-app/assets/6160324/a67f4ff4-e0ae-4dea-a08b-bc6b22988a0d

## List of changes proposed in this pull request
- Updated IDPay specificatins in `package.json`
- Added correct route in button `onPress` handler
- Small refactoring to align the code with the new specifications

## How to test
With the `io-dev-server`, navigate to the beneficiary details screen of an IDPay initiative (the one reachable by pressing the "(i)" button at the top right). Check if the "Preferences & Privacy" button navigate to the service info page.
